src/ansys/tools/visualization_interface/backends/pyvista/pyvista.py

```diff
@@ -412,7 +412,8 @@ def show(
         view_2d: Dict = None,
         name_filter: str = None,
         dark_mode: bool = False,
-        **plotting_options,
+        plotting_options: Optional[Dict[str, Any]] = {},
+        **show_options: Dict[str, Any],
     ) -> List[Any]:
```

Comment:
This is a big breaking change... I'd wish we had discussed about it before merging

Reply:
Pinging @AlejandroFernandezLuces and @SMoraisAnsys to understand what is the reason for this ask

Reply:
@dipinknair pinged me because some users were having issues because of this, they were trying to pass `cpos` to `show` method and nothing was happening. After reviewing the code I though it doesn't make sense to prioritize plotting kwargs over `show` kwargs.

 I think this is the sensible solution, we can discuss offline.

Reply:
Okay - even if it is the sensible solution...

* We should inform people about this change.. this can lead to unexpected behaviors IMO which people might not understand
* I would like that things get reported on GitHub and not just a PR out of nowhere without description... anybody should be able to understand why a decision was made by either reading at the PR description and its associated issue... and none of it happened.

Reply:
Awesome, thanks @AlejandroFernandezLuces for updating it!

Reply:
Oh I really like this one...

Reply:
I wasn't aware of this module, this solution looks good.

In any case, a point to consider for me is, we are allowing to plot and show in the `plot` method. I understand that users that are trying to access PyVista kwargs are "advanced" PyVista users that need customizations. I would argue that these users are familiar with the PyVista usage, understand that show arguments and plot arguments are different and if you are mixing plot and show in the same method you might want to separate both sets of arguments.

I guess it is a matter of simple vs explicit. I'll consider changing the signature back to this approach.

Reply:
Another thing to consider is, that if you have `inspect` approach method, you made the API more "backend" agnostic. Because the filtering of kwargs can happen on a lower level.
For another non-pyvista plotting backend having different `show` and `plot` arguments might not make sense.

However I do totally understand your point about advanced users... and I do not want to influence too much this library with PyMAPDL requirements. So, please do proceed as you think it is the best.

Reply:
Let's make the change then, I've opened an issue for it: